doc/conf.py

```diff
@@ -0,0 +1,223 @@
+# Configuration file for the Sphinx documentation app.
+
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+from datetime import datetime
+import inspect
+import os
+from os.path import relpath, dirname
+import re
+import sys
+
+from pyxem import release_info
+import pyxem
+import requests
+
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+sys.path.append("../")
+
+
+# Project information
+project = "pyxem"
+copyright = release_info.copyright
+author = release_info.author
+version = release_info.version
+release = release_info.version
+
+master_doc = "index"
+
+# Add any Sphinx extension module names here, as strings. They can be extensions
+# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = [
+    "sphinxcontrib.bibtex",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.mathjax",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.linkcode",
+    "sphinx_autodoc_typehints",
+    "sphinx_copybutton",
+    "sphinx_gallery.load_style",
+    "nbsphinx",
+    'sphinx.ext.viewcode',
+]
+
+# Create links to references within kikuchipy's documentation to these packages.
+intersphinx_mapping = {
+    "dask": ("https://docs.dask.org/en/latest", None),
+    "diffpy.structure": ("https://www.diffpy.org/diffpy.structure", None),
+    "diffsims": ("https://diffsims.readthedocs.io/en/latest", None),
+    "hyperspy": ("http://hyperspy.org/hyperspy-doc/current", None),
+    "matplotlib": ("https://matplotlib.org", None),
+    "numpy": ("https://docs.scipy.org/doc/numpy", None),
+    "orix": ("https://orix.readthedocs.io/en/stable", None),
+    "python": ("https://docs.python.org/3", None),
+    "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
+    "skimage": ("https://scikit-image.org/docs/stable", None),
+    "sklearn": ("https://scikit-learn.org/stable", None),
+    "h5py": ("http://docs.h5py.org/en/stable/", None),
+}
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = [
+    "_templates",
+]
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files. This image also affects
+# html_static_path and html_extra_path.
+exclude_patterns = [
+    "_build",
+    "_static/v0.2.0/*.ipynb",
+]
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for a
+# list of builtin themes.
+html_theme = "sphinx_rtd_theme"
```

Comment:
I recommend using [Furo](https://pradyunsg.me/furo/), which is already listed in `setup.py` in this PR.